src/openfermion/ops/_code_operator_test.py

```diff
@@ -0,0 +1,64 @@
+import unittest
+
+from openfermion.ops import SymbolicBinary
+from openfermion.ops._code_operator import BinaryCodeError, \
```

Comment:
I think pep8 would complain about this, right? I think it is preferred to use parentheses in imports like how it is done here for ._fermion_operator: https://github.com/quantumlib/OpenFermion/blob/master/src/openfermion/ops/__init__.py